### PR TITLE
Chore/data cleanup cs

### PR DESCRIPTION
**Pull request by corysanders3**
# Pull Request

## Description
- Add new function called organizeTeas that will remove tea categories
- Function will also add placeholder image for teas without images and then sets the tea state
- Added placeholder favicon

## Testing
- Testing was not done for the organizeTeas function

## Checklist
- [x] The code follows the project's coding standards.
- [ ] Unit tests have been added or updated to cover the changes.
- [x] Documentation has been updated to reflect the changes (if applicable).
- [x] The code compiles without errors.
- [x] The changes have been tested locally and pass all relevant tests.
- [x] All new and existing tests pass.
- [x] The pull request has been reviewed by at least one other contributor.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/leaves2.ico" />
```
Comment (tednaphil) [inline public/index.html:5]: Nice 😎

File: src/components/teas/Teas.tsx [hunk 2]
```diff
@@ -40,7 +42,23 @@ function Teas({ setIsRedirected }: Props) {
 
     function filterTeas(fetchedTeaData: any) {
         const filteredTeaData = fetchedTeaData?.filter((tea: Tea) => tea.type === category)
-        !filteredTeaData.length ? navigate('*', { replace: true }) : setTeas(filteredTeaData);
+        !filteredTeaData.length ? navigate('*', { replace: true }) : organizeTeas(filteredTeaData);
+    }
+
+    function organizeTeas(data: Tea[]) {
```
Comment (tednaphil) [inline src/components/teas/Teas.tsx:48]: Fabulous solution!

**Review by tednaphil — APPROVED** [review]
Thank you for finishing that tea data cleanup!

**Review by EthanDuvall — APPROVED** [review]
Nicely done!

**Pull request merged**